tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Features/Create User.3538.feature

```diff
@@ -21,21 +21,22 @@ Scenario: 1. A authority user can create a user
 	When a POST request is made to create a user for organisation Organisation 2
 		| FirstName | LastName   | PhoneNumber | EmailAddress             |
 		| Bob       | Bobkovitch | 0123456789  | bob.bobkovitch@email.com |
-	Then a response with status code 200 is returned
+	Then a response with status code 201 is returned
```

Comment:
we should probably add something here to verify a user ID is returned

Reply:
Should already have that. Scenario 2 within Create User.3538.feature.